front/lib/api/assistant/generation.ts

```diff
@@ -123,7 +125,12 @@ export async function renderConversationForModelMultiActions({
       messages.push({
         role: "user" as const,
         name: m.context.fullName || m.context.username,
-        content,
+        content: [
```

Comment:
Might break tables query dust app (and maybe other dust apps ?)

We might want to do a pass to ensure everything still works fine there

Reply:
Yes the array of messages can mix both complex type with regular stream: https://github.com/dust-tt/dust/pull/5835.